graph/step.go

```diff
@@ -347,3 +354,12 @@ func addBuildCacheOptsToCmd(domain, path, tag, originalBuildCmd string) (string,
 	}
 	return fmt.Sprintf("--load --cache-to=type=registry,ref=%s,mode=max --cache-from=type=registry,ref=%s %s", cacheImage.String(), cacheImage.String(), originalBuildCmd), nil
 }
+
+func invokesBuildkit(envs []string) bool {
+	for _, env := range envs {
+		if strings.Contains(env, BUILDKIT_ENV_VAR) {
```

Comment:
What will happen for the envs beloow?
- `DOCKER_BUILDKIT=123`
- `CUSTOMIZED_DOCKER_BUILDKIT=1`
- `WHATEVER="DOCKER_BUILDKIT=1"`

Reply:
Updated